internal/services/appservice/service_plan_resource.go

```diff
@@ -100,6 +101,12 @@ func (r ServicePlanResource) Arguments() map[string]*pluginsdk.Schema {
 			Default:  false,
 		},
 
+		"elastic_scale_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			Default:  false,
```

Comment:
This will, unfortunately, be a breaking change for existing EP plan deployments. It has different default values per Plan SKU. It _may_ be possible to handle this in a `CustomiseDiff` function in a non-breaking way. Sorry I did not spot it sooner.